neotest_elixir/neotest_formatter.exs

```diff
@@ -98,12 +93,12 @@ defmodule NeotestElixirFormatter do
   defp make_status(%ExUnit.Test{state: {:excluded, _}}), do: "skipped"
   defp make_status(%ExUnit.Test{state: {:invalid, _}}), do: "failed"
 
-  defp save_test_output(%ExUnit.Test{} = test, config) do
+  defp save_test_output(%ExUnit.Test{} = test, config, id) do
     output = make_output(test, config)
 
     if output do
-      file = Path.join(config.output_dir, "test_output_#{config.test_counter}")
-      File.write!(file, output)
+      file = Path.join(config.output_dir, "test_output_#{:erlang.phash2(id)}")
+      File.write!(file, output, [:append])
```

Comment:
We need a new line or two, multiple failures appear without any separation.

Reply:
```suggestion
      if File.exists?(file) do
        File.write!(file, ["\n\n", output], [:append])
      else
        File.write!(file, output)
      end
```

Reply:
Applied this suggestion 🙇🏻